cmd/geth/log_display_green.go

```diff
@@ -50,30 +51,81 @@ var dominoes = []string{"🁣", "🁤", "🁥", "🁦", "🁭", "🁴", "🁻",
 var chainIcon = "◼⋯⋯" + logger.ColorGreen("◼")
 var forkIcon = "◼⋯⦦" + logger.ColorGreen("◼")
 var headerIcon = "◼⋯⋯" + logger.ColorGreen("❐")
-var downloaderIcon = "◼⋯⋯" + logger.ColorGreen("⬇")
+var downloaderIconStart = "◼⋯⋯" + logger.ColorGreen("⬇")
+var downloaderIconDone = "◼⋯⋯" + logger.ColorGreen("✔︎")
+var downloaderIconFail = "◼⋯⋯" + logger.ColorRed("✕")
 var minedIcon = "◼⋯⋯" + logger.ColorGreen("⟠")
 var lsModeDiscoverSpinners = []string{"➫", "➬", "➭"}
+var downloadingFrom = ""
 
 func greenParenify(s string) string {
 	return logger.ColorGreen("⟪") + s + logger.ColorGreen("⟫")
 }
 func redParenify(s string) string {
 	return logger.ColorRed("⟪") + s + logger.ColorRed("⟫")
 }
+func yellowParenify(s string) string {
+	return logger.ColorYellow("⟪") + s + logger.ColorYellow("⟫")
+}
+func prefix(ev interface{}, e *eth.Ethereum) string {
```

Comment:
... and some minor upgrades to the `--display-green green` format for _prettiness_. And because it helps to establish continuity about downloader starts/fails from given peers.